discretisedfield/region.py

```diff
@@ -364,7 +371,7 @@ def edges(self):
 
         Returns
         -------
-        numpy.ndarray (3,)
+        numpy.ndarray
 
              Edge lengths :math:`(l_{x}, l_{y}, l_{z})`.
```

Comment:
```suggestion
             Edge lengths 3 dimensions :math:`(l_{x}, l_{y}, l_{z})`.
             Edge lengths n dimensions :math:`(l_{0}, ..., l_{n})`.
```

What do you think of something like this?
We could add the default n dimension names.

If we wanted something like this I could go through and edit all the doc strings with this example

Reply:
I like the the `n` dimensions idea. If you like, please go ahead and edit it everywhere.